osidb_bindings/bindings/python_client/models/audit.py

```diff
@@ -190,10 +163,10 @@ def get_fields():
             "pgh_slug": str,
             "pgh_obj_model": str,
             "pgh_label": str,
-            "pgh_diff": AuditPghDiff,
```

Comment:
I wonder why are we losing the types here

Reply:
After [inspecting the openapi](https://github.com/RedHatProductSecurity/osidb-bindings/pull/58/files#diff-3fc458d150a54a5d8e7648ebc45c88f7a470b95f71ac66c359ad33f227acb976L7526-L7535) file (you need to show the whole diff of the openapi file as it is hidden) it seems that the `pgh_diff` and `pgh_context` used to be objects and now they are not according to API schema and `pgh_data` never was of type object. Checking it against the actual API result from OSIDB I see that all these three are still objects. I think this is some discrepancy caused by the Django update. I think there is not much we can do about it here, this should be properly fixed in the OSIDB repository. I will file a bug for that. A good thing is that history audit is currently not exposed via bindings so it should not impact any users. Wdyt?

Reply:
If the history API is not exposed to the users then I don't see any problem with it but I agree that this should be at least looked up on OSIDB